### DISCUSSION TITLE
DaMaRiS boundary conditions

### DISCUSSION THREAD

**Discussion by n-desjardins** [topas-nbio/TOPAS-nBio#20] [Simulation-question]
### Simulation question

Hi all,

I was wondering why the reflective boundary conditions in DaMaRiS were set to reflect particles that strictly land on the boundary as opposed to particles that land on or outside the boundary (== vs >=) . It appears to allow particles to jump over the boundary and “get lost”. 

I originally posted this question on the Google Group but decided to move it here. Also I was not sure what "TOPAS-nBio category" to pick. Would a "DaMaRiS" category make sense? 

Thank you,
Nicolas
[
<img width="892" alt="DaMaRiS_BC" src="https://github.com/user-attachments/assets/3f27509e-48c3-49c1-ae47-3932aa665d3b">
](url)

### TOPAS-nBio category

Geometry

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment]
I think that is always a possibility with Brownian motion which can partially be mitigated by reducing the time step. I think you can control that using the parameter `BoundingCellOrNucleusRadius` Or have you tried comparing sims using == vs >= to see how sensitive are the results?

**Comment by JohnWilliamWarmenhoven** [comment]
Dear Nicholas,

I believe the way the stepper works is that it won't allow a "particle" to step over the boundary of a geometry but will instead place it on the boundary if it were to attempt that. I've not noticed any of the DNA "particles" going missing before, are you seeing this in your simulations?

Kind regards,
John-William
